### ISSUE TITLE
some questions about training 

### ISSUE THREAD

**Issue by yanqian123** [yjxiong/temporal-segment-networks#229]
I am a new about this field, and have a question when i was training the network. I just have one GPU, so when i was training,i set the batch_size=12 ,in order to meet the condition about "batch_size*iter_size*n_gpu=128", i set iter_size=11, so when i was training ,the log showed "Gradient clipping: scaling down gradients (L2 norm 68.3038 > 40) by scale factor 0.585619" . i want to know, if it will make a big influence about the result. thanks a lot and i am looking forward to your reply if you can give me some advice

**Comment by yjxiong** [comment]
It is fine as long as you can get a good final accuracy. But generally, I would recommend getting more GPUs as training with 1GPU will be very slow.

**Comment by Mohamad73** [comment]
how did u change the iteration size

**Comment by yanqian123** [comment]
I  only used one GPU ,i set iters more than four times，the result is the same as the paper shown

**Comment by Mohamad73** [comment]
I meant from which file? I could not find the file where you can change the iter_size.
Thank you for your help

**Comment by yanqian123** [comment]
about temporal-segment-networks-master/models/ucf101/tsn_bn_inception_flow_solver.prototxt , you can change  max_iter=....

**Comment by yjxiong** [comment]
@Mohamad73 It is in the solver file. The option is named `iter_size`
